src/components/Layout/Header/GlobalSearch/StakingResults/StakingResult.tsx

```diff
@@ -0,0 +1,88 @@
+import { Flex, forwardRef } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { Amount } from 'components/Amount/Amount'
+import { AssetCell } from 'components/StakingVaults/Cells'
+import { RawText } from 'components/Text'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import type { OpportunityId } from 'state/slices/opportunitiesSlice/types'
+import type { GlobalSearchResult } from 'state/slices/search-selectors'
+import { GlobalSearchResultType } from 'state/slices/search-selectors'
+import {
+  selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
+  selectAssetById,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { ResultButton } from '../ResultButton'
+
+type StakingResultProps = {
+  stakingId: OpportunityId
+  index: number
+  activeIndex?: number
+  onClick: (arg: GlobalSearchResult) => void
+}
+
+export const StakingResult = forwardRef<StakingResultProps, 'div'>(
+  ({ stakingId, onClick, index, activeIndex }, ref) => {
+    const stakingOpportunities = useAppSelector(
+      selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
+    )
+    const opportunity = stakingOpportunities.find(opportunity => opportunity.id === stakingId)
```

Comment:
this is going to return a new ref every time - should be able to select by staking id